daemon/qrexec-client.c

```diff
@@ -40,6 +40,10 @@ int replace_chars_stderr = 0;
 
 #define VCHAN_BUFFER_SIZE 65536
 
+#if MAX_DATA_CHUNK_V3 < MAX_DATA_CHUNK_V2
+#error "MAX_DATA_CHUNK_V3 must not be smaller than MAX_DATA_CHUNK_V2"
+#endif
```

Comment:
Do wee need this check twice? Maybe move it into the header?

Reply:
I wanted to have it explicitly in `.c` file to guard against changing `.h` without considering all the files using it. Otherwise I'd add just a comment.

Reply:
I personally would have added the assert in the header (maybe with a comment where/why this assumption is used/needed). Not only a comment such that the compiler catches a change even if someone didn't read the comment.

Anyway, your version is also reasonable.

As above maybe add a short comment.